pkg/kubectl/cmd/run.go

```diff
@@ -34,7 +35,7 @@ import (
 
 const (
 	run_long = `Create and run a particular image, possibly replicated.
-Creates a replication controller to manage the created container(s).`
+Creates a deployment or job to manage the created container(s).`
```

Comment:
Talked with @bgrant0607 about this. The users should specify a generator when they want to depend on a specific version of the generator (for example, `run/v1`). Since job is beta I'll keep it as default; but deployment isn't beta yet, so keep rc as default.